spacepy/pybats/bats.py

```diff
@@ -640,40 +646,122 @@ def trace(self, bats):
 
         # Check if line is closed to body.
         if 'rbody' in bats.attrs:
-            r1 = sqrt(self.x[0]**2.0  + self.y[0]**2.0)
+            r1 = sqrt(self.x[ 0]**2.0 + self.y[0]**2.0)
             r2 = sqrt(self.x[-1]**2.0 + self.y[-1]**2.0)
             if (r1 < bats.attrs['rbody']) and (r2 < bats.attrs['rbody']):
                 self.open = False
 
     def plot(self, ax, *args, **kwargs):
         '''
-        Add streamline to axes object "ax". 
+        Add streamline to axes object "ax".
         '''
         ax.plot(self.x, self.y, self.style, *args, **kwargs)
 
+
 class Bats2d(IdlFile):
     '''
-    A child class of :class:`~pybats.IdlFile` taylored to BATS-R-US output.
+    A child class of :class:`~pybats.IdlFile` tailored to 2D BATS-R-US output.
+
+    Calculations
+    ------------
+    
```

Comment:
Same question about whether blank lines under section headings affect the rendering in docs

Reply:
Was this one being left as-is or did you want to amend your last commit and force push?

Reply:
Derp.
Fixed.